L1Trigger/L1TCommon/src/Parameter.cc

```diff
@@ -0,0 +1,115 @@
+#include "L1Trigger/L1TCommon/interface/Parameter.h"
+
+using namespace std;
+
+namespace l1t{
+
+Parameter::Parameter(const char *id,
+                     const char *procOrRole,
+                     const char *type,
+                     const char *value,
+                     const char *delimeter
+                    ){
+    this->id = id;
```

Comment:
would be more nicely handled by following a naming convention. Eg, a usual one in CMS is member data ends in '_'